Backpack/Calendar/Classes/BPKCalendar.m

```diff
@@ -1,5 +1,155 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 #import "BPKCalendar.h"
+#import "BPKCalendarHeaderCell.h"
+#import "BPKCalendarStickyHeader.h"
+#import "BPKCalendarAppearance.h"
+
+#import <Backpack/Color.h>
+#import <Backpack/Font.h>
+#import <Backpack/Spacing.h>
+#import <FSCalendar/FSCalendar.h>
+#import <FSCalendar/FSCalendarCollectionView.h>
+#import <FSCalendar/FSCalendarDynamicHeader.h>
+#import <FSCalendar/FSCalendarWeekdayView.h>
+#import <Foundation/Foundation.h>
+#import <objc/runtime.h>
+
+@interface FSCalendar (Private)
+
+@property (weak, nonatomic) FSCalendarCollectionView *collectionView;
```

Comment:
Why is this not in the empty category below?

Reply:
We need to be able to access the collectionView property of FSCalendar to be able to replace the header cell with our own subclass.

Reply:
Okay fair enough